### ISSUE TITLE
docs: add hardware pre-requisites

### ISSUE THREAD

**Issue by fitzthum** [confidential-containers/confidentialcontainers.org#50]
The documentation now has a getting started guide with a hw prerequisites section. I need help from hw vendors to fill out these pages. It should be relatively easy.

The idea of each hw section is to explain anything a user would need to do before they will be able to use CoCo. This could include which machines are supported in the first place, fw configurations, host software stack (where do they get the special kernel), etc.

It's up to individual vendors how they want to handle this. The most important thing is to make sure that an inexperienced user will be able to understand what the steps are and follow them. You can link to external pages for certain steps or include everything here. We already have material in the github that we can pull from. Once the quickstart guide is complete here we will remove it from the github, so make sure all the information you want is here. Please don't include anything about setting up the operator or running workloads. That will come in later, generic, sections, which won't need to be owned by any particular vendor.

Who can help out with this?

AMD: @arvindskumar99 @ryansavino?
Intel: @mythi @dcmiddle?
SE: @BbolroC?

**Comment by ryansavino** [comment]
@arvindskumar99 will take the initial stab, and @AdithyaKrishnan and I will help as well from the AMD side. Thanks @fitzthum.

**Comment by mythi** [comment]
> Who can help out with this?

I don't have a name yet for the Intel stuff but we'll work on our part to the website.

**Comment by BbolroC** [comment]
Of course, I am happy to help with anything you need for IBM SE. Thanks.
